docs/reference/index.rst

```diff
@@ -6,17 +6,19 @@ Reference
 .. toctree::
    :maxdepth: 2
 
-   simulations
-   models
+   align
```

Comment:
I couldn't really make sense of the ordering in this toctree so I made it alphabetical.  If we want to go back to the way it was that is fine, I just thought it might have been a legacy thing with no real good reason

Reply:
you are correct and I have no issue with this